smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/i18n/KafkaLogging.java

```diff
@@ -175,14 +167,10 @@ void waitingForAckForTooLong(long offset, TopicPartition topicPartition, long de
     @Message(id = 18236, value = "Will mark throttled commit strategy for group '%s' as unhealthy if records go more than %d milliseconds without being processed.")
     void setThrottledCommitStrategyReceivedRecordMaxAge(String group, long unprocessedRecordMaxAge);
 
-    @LogMessage(level = Logger.Level.INFO)
+    @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 18237, value = "Setting client.id for Kafka producer to %s")
     void setKafkaProducerClientId(String name);
 
-    @LogMessage(level = Logger.Level.INFO)
-    @Message(id = 18238, value = "Setting client.id for Kafka consumer to %s")
-    void setKafkaConsumerClientId(String name);
```

Comment:
Actually it'd be nice log the consumer client id's somewhere here : https://github.com/smallrye/smallrye-reactive-messaging/blob/5296684541b3bcad5bdfc65c3db4666a0350c3e3/smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/ReactiveKafkaConsumer.java#L221

Reply:
I'm actually testing it with Quarkus and add a few more condensed log messages (like we lost the bootstrap server too).